homeassistant/components/ohme/button.py

```diff
@@ -0,0 +1,70 @@
+"""Platform for button."""
+
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+
+from ohme import ChargerStatus, OhmeApiClient
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import OhmeConfigEntry
+from .entity import OhmeEntity
+
+PARALLEL_UPDATES = 0
```

Comment:
The coordinator only parallelizes data updates not outgoing actions, can the chargers really deal with parallel actions on button presses?

Reply:
Unlikely - so set this to 1?

Reply:
Also, it would be nice to update the coordinator after this button press, whats the best way to do that?

Reply:
Yes 1. `await self.coordinator.async_request_refresh()` schedules an update